src/components/sidebar-entities/settings-sidebar/index.js

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+import { injectIntl } from 'react-intl';
+
+import { updateSettings } from 'store/actions/settings.actions';
+
+import SettingsSidebar from './settings-sidebar';
+
+const mapStateToProps = state => ({
+  settings: state.settings,
+});
+
+const mapDispatchToProps = dispatch => ({
+  updateSettings: (key, value) => dispatch(updateSettings(key, value)),
+});
+
+export default injectIntl(
+  connect(
+    mapStateToProps,
+    mapDispatchToProps,
```

Comment:
Instead of defining a `mapDispatchToProps` function, you could just pass an object since the incoming parameters are the same as the parameters defined on `updateSettings`.

```
const mapDispatchToProps = { updateSettings };
```

Reply:
Fixed in Irame/sectors-without-number@3e8c22179164e4aa335f4a4e3daa4b3a158d39d8